### DISCUSSION TITLE
Task 3

### DISCUSSION THREAD

**Discussion by mmh62** [Part-II-Computational-Physics/.github#51] [General]
Pretty sure code is correct but getting: 

No renderer found for "image/png"

**Comment by JNHallen** [comment]
That sounds like it could be an issue with your installation of jupyter. Have you been able to plot the other figures?

**Comment by mmh62** [comment]
no, I haven't been able to, I have all the Jupiter extensions downloaded tho...
________________________________

**Reply by JNHallen** [reply]
I would try opening a blank jupyter notebook and just putting in this code:

```
import matplotlib.pyplot as plt

plt.figure()
plt.plot([1, 2, 3], [1, 2, 3])

```

and see if that plots okay.

**Reply by mmh62** [reply]
it works if I put this in but it is still not working on the actual repo

**Reply by JNHallen** [reply]
I will send you an email about this.
